cli/src/main.rs

```diff
@@ -247,29 +346,83 @@ struct Search {
     offset: Option<usize>,
     #[structopt(short, long)]
     limit: Option<usize>,
+    #[structopt(short, long, conflicts_with_all=&["query", "filter", "offset", "limit"])]
```

Comment:
```suggestion
    #[structopt(short, long, conflicts_with = "query")]
```

Why would it be invalid to specify filters, offsets, or limits?

Reply:
Because it is not yet implemented

Reply:
My goal was to enable it from the interactive prompt instead, so those will always be ignored